### ISSUE TITLE
[TW-1914] Bug. Problem with UTF description and annotations.

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1927]
_idontknow on 2017-06-04T17:42:41Z says:_ 

 *How to reproduce:*
task add АБВ
task 1 annotate some annotation
task next

*What happened:*
corrupted output of description (А�)

*What I expected:*
correct output of description (АБВ)

*You can use this texts:*
АБВ becomes А�
你好 becomes 你� 

*diagnostics command output:*
task 2.5.1
   Platform: Linux

Compiler
    Version: 6.2.1 20161212
       Caps: +stdc +stdc_hosted +LP64 +c8 +i32 +l64 +vp64 +time_t64
 Compliance: C++11

Build Features
      CMake: 3.7.1
    libuuid: libuuid + uuid_unparse_lower
  libgnutls: 3.5.6
 Build type: None

Configuration
       File: /home/abcdef/.taskrc (found), 1402 bytes, mode 100664
       Data: /home/abcdef/SyncData/.task (found), dir, mode 40755
    Locking: Enabled
         GC: Enabled
     Server: 
      Trust: strict
Certificate: , not readable, 0 bytes
        Key: , not readable, 0 bytes
    Ciphers: NORMAL
      Creds: 

Hooks
     System: Enabled
   Location: /home/abcdef/SyncData/.task/hooks
             (-none-)

Tests
      $TERM: xterm-256color (206x111)
       Dups: Scanned 4 tasks for duplicate UUIDs:
             No duplicates found
 Broken ref: Scanned 4 tasks for broken references:
             No broken references found

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2017-06-04T17:42:41Z
Modified: 2017-06-06T23:40:01Z
```

**Comment by taskwarrior** [comment]
_idontknow on 2017-06-04T17:49:34Z says:_ 

 It's really annoying problem.

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2017-06-06T23:40:01Z says:_ 

 Another duplicate of TW-1778, TW-1823 and TW-1902.  This is fixed in 2.6.0.

**Comment by pbeckingham** [comment]
![tw-1914_screenshot](https://user-images.githubusercontent.com/7963/36643732-cce34296-1a1d-11e8-8858-a3da20268d9c.png)
